apps/remote-processor-service/Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Repo name: arynai/sycamore-remote-processor-service
+
+##########
+# Common: resolve dependencies
+FROM python:3.11 AS rps_common
+
+ARG RPS_PORT=2796
+ARG POETRY_NO_INTERACTION=1
+ARG POETRY_VIRTUALENVS_IN_PROJECT=1
+ARG POETRY_VIRTUALENVS_CREATE=1
+ARG POETRY_CACHE_DIR=/tmp/poetry_cache
+
+# Is there some way to keep this common layer common across all our services?
+# E.g. maybe we can have an image called 'aryn_service_base' or something
+# - setup aryn user and directory
+# - install commonly used software (poetry, maybe protoc)
+# And then we can just layer services on top?
+
+WORKDIR /aryn/
+COPY ./apps/remote-processor-service/Makefile ./
+RUN make aryn_user
+RUN rm -f /etc/apt/apt.conf.d/docker-clean; echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache
+RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
```

Comment:
Later, we should probably split this into Dockerfile.buildx (this version) and Dockerfile.build, which works with vanilla docker build.  The mount and chown stuff is buildx only.